conf/gobblin-env.sh

```diff
@@ -0,0 +1,9 @@
+#!/bin/bash
+
+# Set Gobblin specific environment variables here.
+
+#Gobblin work dir
+export GOBBLIN_WORK_DIR=hdfs://localhost/gobblin
```

Comment:
Fixed. I suspect that ${env:GOBBLIN_WORK_DIR} could be probably eliminated from gobblin-mapreduce.properties. What about having
fs.uri=hdfs://localhost:8020
gobblin.work.dir=${fs.uri}/gobblin 
instead?